rum-demo-app-ios/RUM Mobile Demo App/Common/AppDelegate.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  AppDelegate.swift
+//  RUM Mobile Demo App
+//
+//  Created by Akash Patel on 06/01/22.
+//
+
+import UIKit
+import CoreLocation
+import SplunkOtel
+import SplunkOtelCrashReporting
+import IQKeyboardManager
+
+
+let config = Configuration()
+let mainStoryBoard = UIStoryboard(name: "Main", bundle: nil)
+let screenSize: CGRect = UIScreen.main.bounds
+let screenWidth = screenSize.width
+let screenHeight = screenSize.height
+
+var coordinate = CLLocationCoordinate2D()
+var placemarkDescription = CLPlacemark()
+
+let geoCoder = CLGeocoder()
+let sharedCart = Cart.sharedInstance  //shared instance of cart class
+
+var timerForLocation : Timer?
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+    let locationObj = CLLocationManager()
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        window?.backgroundColor = .white
+        IQKeyboardManager.shared().isEnabled = true
+       
+        locationObj.delegate = self
+        locationObj.requestWhenInUseAuthorization()
+        
+        if #available(iOS 14.0, *) {
+            if locationObj.authorizationStatus == CLAuthorizationStatus.authorizedWhenInUse {
+                locationObj.startUpdatingLocation()
+            }
+        } else {
+            // Fallback on earlier versions
+            if CLLocationManager.authorizationStatus() == CLAuthorizationStatus.authorizedWhenInUse {
+                locationObj.startUpdatingLocation()
+            }
+        }
+                
+    
+        SplunkRum.initialize(beaconUrl: AppVariables.current.beaconURL , rumAuth: AppVariables.current.token ,options: SplunkRumOptions(debug: true, environment: AppVariables.current.environment))
+            
+            SplunkRumCrashReporting.start()
+            SplunkRum.setGlobalAttributes(["DeviceID": UIDevice.current.identifierForVendor?.uuidString as Any])
+            //https://ingest.us1.signalfx.com  -- realm URL
```

Comment:
@akashCrest Let's remove the commented block of code

Reply:
Removed commented block.